src/audio/google/google_rtc_audio_processing.c

```diff
@@ -453,7 +449,36 @@ static int google_rtc_audio_processing_init(struct processing_module *mod)
 		comp_err(dev, "Failed to apply GoogleRtcAudioProcessingParameters");
 		goto fail;
 	}
+#if CONFIG_IPC_MAJOR_4
+	size_t buf_size = cd->num_frames * cd->num_capture_channels * sizeof(cd->process_buffer[0]);
 
+	comp_dbg(dev, "Allocating process_buffer of size %u", buf_size);
+	cd->process_buffer = rballoc(0, SOF_MEM_CAPS_RAM, buf_size);
+	if (!cd->process_buffer) {
+		comp_err(dev, "Allocating process_buffer failure");
+		ret = -EINVAL;
+		goto fail;
+	}
+	bzero(cd->process_buffer, buf_size);
+	buf_size = cd->num_frames * sizeof(cd->aec_reference_buffer[0]) *
+			cd->num_aec_reference_channels;
+	comp_dbg(dev, "Allocating aec_reference_buffer of size %u", buf_size);
+	cd->aec_reference_buffer = rballoc(0, SOF_MEM_CAPS_RAM, buf_size);
```

Comment:
nit picky here: Would be nice to have the computation ` cd->num_frames * sizeof(cd->aec_reference_buffer[0]) * cd->num_aec_reference_channels` instead of `buf_size` as the buffer size might be different for render and for capture.

Reply:
Sorry I must have been tired when I did the review. It looks good as it is.